src/pages/billing/NewBillingProjectWizard/StepWizard/StepHeader.ts

```diff
@@ -1,14 +1,13 @@
+import { PropsWithChildren } from 'react'
 import { h3, header, span } from 'react-hyperscript-helpers'
 
 
-interface StepHeaderProps {
+type StepHeaderProps = PropsWithChildren<{
   title: React.ReactNode
-  children?: React.ReactNode[]
-  styles?: React.CSSProperties
-}
+  style?: React.CSSProperties
```

Comment:
For consistency, use `style` instead of `styles`.

Reply:
Also, it looks like the style prop for some of these components is never used and could be removed entirely.

Reply:
It may be that they are used in the Azure billing project usage that is still under development.